metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/lineage.py

```diff
@@ -256,11 +256,14 @@ def get_lineage_workunits(
         sql_parser_schema_resolver: SchemaResolver,
         view_refs_by_project: Dict[str, Set[str]],
         view_definitions: FileBackedDict[str],
+        snapshot_refs_by_project: Dict[str, Set[str]],
+        snapshot_definitions: FileBackedDict[str],
         table_refs: Set[str],
     ) -> Iterable[MetadataWorkUnit]:
         if not self._should_ingest_lineage():
             return
         views_skip_audit_log_lineage: Set[str] = set()
+        snapshots_skip_audit_log_lineage: Set[str] = set()
```

Comment:
do we need both views_skip_audit_log_lineage and snapshots_skip_audit_log_lineage? seems like they can be merged into a single variable here